src/Forms/Form.php

```diff
@@ -339,7 +339,7 @@ public function requireSudoMode(): void
         // Check that the current request handler for the form is one that's used
         // in an admin context where sudo mode makes sense
         $classes = [
-            LeftAndMain::class,
+            AdminController::class,
```

Comment:
Also update reference to `LeftAndMain` to `AdminController` in the PHPDoc.

Reply:
Updated